### DISCUSSION TITLE
How do you use "dotnet test --filter" to filter tests by an attribute with more than one parameter?

### DISCUSSION THREAD

**Discussion by japarson** [xunit/xunit#2239] [Question] [answered]
My tests use an attribute that accepts two parameters:

[Test]
[Platform(PlatformIdentifier.UAP, PlatformVersion.LatestVersion)]
MyTest()
{
}

The Platform class derives from System.Attribute and PlatformIdentifer/PlatformVersion are enums.

My problem is that I'm struggling to draft a test filter string that can somehow use this attribute. I've tried the following strings to no avail:
- (Platform=UAP.LatestVersion)
- (Platform=UAP LatestVersion)
- (Platform=UAP)

How can I correctly form a test filter string to query tests with this attribute?

**Comment by bradwilson** [comment, marked as answer]
Are you sure you're in the right place? `[Test]` and `[Platform]` aren't xUnit.net attributes. :)

**Reply by japarson** [reply]
Ah, this would be an nunit question then. Thank you!
